c10/util/string_view.h

```diff
@@ -488,7 +488,7 @@ class basic_string_view final {
       const noexcept {
     if (pos + 1 <= size()) {
       for (size_type cur = pos; cur < size(); ++cur) {
-        if (condition(at_(cur))) {
+        if (std::forward<Condition>(condition)(at_(cur))) {
```

Comment:
This smells like a false positive? How can something be forwarded again, and again in a loop? Wouldn't that cause a use-after-move bug?

Reply:
Oh, it should be a false positive